File: PAT (Basic-Level)/C/1066.c
#include <stdio.h>

int main(int argc, char const *argv[]){
    int M, N, A, B, replaceNum;
    scanf("%d %d %d %d %d", &M, &N, &A, &B, &replaceNum);
    int temp;
    for (int i = 0; i < M; i++){
        for (int j = 0; j < N; j++){
            scanf("%d", &temp);
            if(temp >= A && temp <= B)
                temp = replaceNum;
            printf("%03d", temp);
            if(j == N-1)
                printf("\n");
            else 
                printf(" ");
        }
    }
    return 0;
}

File: PAT (Basic-Level)/Python/1066.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-
M, N, A, B, replaceNum = input().split()
M = int(M)
N = int(N)
A = int(A)
B = int(B)
replaceNum = int(replaceNum)

for i in range(M):
    data = list(map(int, input().split()))
    for j in range(N):
        temp = data[j]
        if data[j] >= A and data[j] <= B:
            temp = replaceNum
        print("%03d" % temp, end = '')
        if j == N-1:
            print()
        else:
            print(" ", end = '')

#testpoint3 overtime